event_routing_backends/management/commands/transform_tracking_logs.py

```diff
@@ -159,6 +160,25 @@ def validate_destination(driver, container_name, prefix, source_objects):
     print(f"Wrote source file list to '{container_name}/{full_path}'")
 
 
+def validate_lrs_routes(lrs_urls):
+    """
+    Validate that the provided LRS URLs are present and enabled in the RouterConfiguration.
+
+    Raises a ValueError if any of the URLs are missing or not enabled.
+    """
+    if lrs_urls:
+        missing_urls = set(lrs_urls) - set(
+            RouterConfiguration.objects.filter(
+                route_url__in=lrs_urls, enabled=True
+            ).values_list("route_url", flat=True)
```

Comment:
I think we should simplify this to something like

```suggestion
        if not RouterConfiguration.objects.filter(
                route_url__in=lrs_urls, enabled=True
            ).exists():
            raise ValueError(
                "The following LRS URLs are not present or not enabled in the ",
                f"RouterConfiguration: {', '.join(missing_urls)}"
            )
```

Reply:
I think this will cause an issue. For example, if there are three enabled routers and a user wants to send data to two of them but misspells one in the management command, the condition `RouterConfiguration.objects.filter(..).exists()` will still return true. In the best case, we should inform the user about the invalid route_urls.